libs/components/modals/src/lib/modules/modal/modal.service.spec.ts

```diff
@@ -176,9 +176,137 @@ describe('Modal service', () => {
 
   it('should show a modal with a wrapper class', fakeAsync(() => {
     const wrapperClass = 'custom-wrapper-class';
-    openModal(ModalTestComponent, { wrapperClass });
+    const modal = openModal(ModalTestComponent, { wrapperClass });
     applicationRef.tick();
 
     expect(document.body.querySelector(`sky-modal.${wrapperClass}`)).toExist();
+    closeModal(modal);
+  }));
+
+  it('should not hide modal host from screen readers when opening a modal', fakeAsync(() => {
+    const modal = openModal(ModalTestComponent, { fullPage: false });
+
+    expect(
+      document.querySelector('sky-modal-host').getAttribute('aria-hidden')
+    ).toBe(null);
+    closeModal(modal);
+  }));
+
+  it('should hide and unhide elements at the host level from screen readers', fakeAsync(() => {
+    const div = document.createElement('div');
+    document.body.appendChild(div);
+    const modal = openModal(ModalTestComponent, { fullPage: false });
+
+    expect(div.getAttribute('aria-hidden')).toBe('true');
+
+    closeModal(modal);
+
+    expect(div.getAttribute('aria-hidden')).toBe(null);
+    div.remove();
+  }));
+
+  it('should keep hidden status elements at the host level consistent', fakeAsync(() => {
+    const div = document.createElement('div');
+    document.body.appendChild(div);
+    div.setAttribute('aria-hidden', 'true');
+
+    const modal = openModal(ModalTestComponent, { fullPage: false });
+
+    expect(div.getAttribute('aria-hidden')).toBe('true');
+
+    closeModal(modal);
+
+    expect(div.getAttribute('aria-hidden')).toBe('true');
+    div.remove();
+  }));
+
+  it('should not hide host siblings that are live', fakeAsync(() => {
+    const div = document.createElement('div');
+    document.body.appendChild(div);
+    div.setAttribute('aria-live', 'true');
+
+    openModal(ModalTestComponent, { fullPage: false });
+
+    expect(div.getAttribute('aria-hidden')).toBe(null);
+    div.remove();
+  }));
+
+  it('what does it do with layered modals', fakeAsync(() => {
+    openModal(ModalTestComponent, { fullPage: false });
+    document.querySelector('sky-test-cmp').id = 'sibling';
```

Comment:
Can we add more descriptive IDs than "sibling" and "lowerModal"? I was having trouble tracking the logic of the test with those names. Also, the "lowerModal" is opened after the "sibling" modal; shouldn't it be the "upperModal" instead?